.github/workflows/ci.yml

```diff
@@ -81,6 +81,14 @@ jobs:
       - name: Run cargo fmt
         run: cargo fmt --all --check
 
+      - name: Install Taplo
+        uses: uncenter/setup-taplo@v1
```

Comment:
I'm not a huge fan of using another third party actions provider, but I see that wgpu also uses this.

It doesn't seem like the maintainer of this action actually uses it, which is slightly concerning.
If we just depended on a specific hash rather than just `@v1`, the security concern goes away

Reply:
For reference, this wouldn't have been my immediate reaction if we used the [`taiki-e/install-action`](https://github.com/taiki-e/install-action) here.

Reply:
I just looked into adding taplo to the install-action list, but the format of taplo's release artifacts seems to change from release to release, so I'm not comfortable taking on that work.

Using a specific hash for uncenter's action seems easy enough.